.changeset/strong-pianos-hug.md

```diff
@@ -0,0 +1,10 @@
+---
+"@khanacademy/wonder-blocks-accordion": patch
+"@khanacademy/wonder-blocks-clickable": patch
+"@khanacademy/wonder-blocks-dropdown": patch
+"@khanacademy/wonder-blocks-popover": patch
+"@khanacademy/wonder-blocks-modal": patch
+"@khanacademy/wonder-blocks-link": patch
+---
+
+Tooling: Enabled jsx-a11y lint rules and disabled existing errors that were found
```

Comment:
There were no changes to the functionality in these packages since we only added `eslint-disable-next-line` to those files. Needed to add a changeset file to address this [ci error](https://github.com/Khan/wonder-blocks/actions/runs/13273401396/job/37057889431)!

Reply:
FWIW `pnpm changeset --empty` could work as well as all the JS code comments are stripped from bundles, but it is fine if you want to keep this as a patch version.

Reply:
I was able to use an empty changeset instead!